modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -6,12 +6,13 @@ You can use a `SharedSecret` object to securely access the entitlement keys of a
 
 The `SharedSecret` object allows you to share and synchronize secrets across namespaces.
 
-:FeatureName: Shared Resource CSI Driver
-include::snippets/technology-preview.adoc[]
+[NOTE]
+====
+The Shared Resource CSI Driver feature is now generally available and included in Builds 1.1 or later for OpenShift. To use this feature, ensure you are running Build 1.1 or a more recent version.
```

Comment:
IMO this will be confusing to customers. I read this initially as "this is GA in OpenShift itself". We need to be very clear that this will be a _deprecated_ feature in OpenShift core, and will be GA in a separate, layered product.